contrib/podmanimage/machine/subuid

```diff
@@ -0,0 +1 @@
+core:100000:1000000
```

Comment:
Hmm, is there really no way to tweak this on the `useradd` commandline?  That'd work better...

Reply:
None that I found.